index.html

```diff
@@ -38,146 +38,138 @@
-      <div class="dwyl-bg-mint white">
-        <h2 class="pv3 text-shadow f2 mv0 tc pv4">Training &amp;<br/> Workshops</h2>
-      </div>
-      <div class="pt2 center dwyl-dark-gray mw8 mt4 ph4 ph0-l">
-        <h3 class="fw9 f3 w-80 w-100-ns center tl">Hands-on learning to Empower and Inspire.</h3>
-        <p class="w-100-ns center tl f5 b">Join one of our masterclasses or tailor a class to suit your needs</p>
-        <ul class="b pl3 center tl" style="font-size: 0;">
-          <div class="dib-ns w-third-ns f5 v-top">
-            <li class="mb1 pr2">Agile Development Methodology</li>
-            <li class="mb1 pr2">Customer Development</li>
-            <li class="mb1 pr2">Cyber Security</li>
-          </div>
-          <div class="dib-ns w-third-ns f5 v-top">
-            <li class="mb1 pr2">User Testing</li>
-            <li class="mb1 pr2">Product Ownership</li>
-            <li class="mb1 pr2">Advanced Analytics</li>
-          </div>
-          <div class="dib-ns w-third-ns f5 v-top">
-            <li class="mb1 pr2">Human-centered Design</li>
-            <li class="mb1 pr2">Code</li>
-            <li class="mb1 pr2">Data Visualisation</li>
-          </div>
-        </ul>
-      </div>
-      <div class="tc mt4-ns center mb5 mb5-l">
-        <a href="#contact" class="dib center link mt2 ph4 pv2 dwyl-bg-mint white br2 f5 shadow-4">Enquire Now</a>
+    <section class="w-100 dwyl-bg-dark-gray tc tl-ns mw8 center pt4 ph5">
+      <div class="mw6-5 center">
+        <h2 class="white text-shadow f2 mv0 pt4">Training &amp; Workshops</h2>
+        <div class="dwyl-dark-gray">
+          <h3 class="fw9 f3 dwyl-yellow center tl mb0 m5">Hands-on learning to Empower and Inspire.</h3>
+          <p class="w-100-ns center tl f5 white b mt2">Join one of our masterclasses or tailor a class to suit your needs</p>
+          <ul class="b center tl pl0 mt3" style="font-size: 0;">
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Customer Development</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Cyber Security</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">User Testing</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Product Ownership</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Advanced Analytics</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Human-centered Design</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Agile Development Methodology</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Code</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Data Visualisation</li>
+          </ul>
+        </div>
+        <a href="#contact" class="dib center link mt2 mb5 ph4 pv2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">Enquire Now</a>
```

Comment:
The lack of padding at the bottom of this section _feels_ like a bug to me:
<img width="1246" alt="screen shot 2017-10-09 at 19 13 15" src="https://user-images.githubusercontent.com/4185328/31352278-10e48a6e-ad26-11e7-9bbb-e83ccbefafb4.png">

I know this is what is in the designs (as per the last comment here: https://github.com/dwyl/dwyl-site/issues/407#issue-262374745) but the lack of symmetry between the white padding at the top and the abrupt end of this section is completely lost on me.

Would you agree @markwilliamfirth @finnhodgkin ?

Reply:
These changes are implementing the designs that we accepted, so this question is probably for @harrygfox

Reply:
I haven't changed the bottom padding, but the white margins don't exist anymore and I think it makes the bottom look less weird.